lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -687,7 +699,7 @@ def main():
             load_config(module, cmds)
             results['changed'] = True
             if module.params['interface']:
-                if delta.get('mode'): # or delta.get('admin_state'):
+                if delta.get('mode'):  # or delta.get('admin_state'):
```

Comment:
Just drop the comment, it doesn't seem to be helpful and `admin_state` gets checked on the very next line of code